thor/thor.php

```diff
@@ -181,21 +181,23 @@ function append_thor_elements_to_form(&$disco_obj, $include_submit = true)
 		{
 			foreach ($xml->document->tagChildren as $node)
 			{
-				// echo "running on [" . $node->tagName . "]...<br>";
-				if ($node->tagName == 'input') $this->_transform_input($node, $disco_obj);
-				elseif ($node->tagName == 'date') $this->_transform_date($node, $disco_obj);
-				elseif ($node->tagName == 'time') $this->_transform_time($node, $disco_obj);
-				elseif ($node->tagName == 'textarea') $this->_transform_textarea($node, $disco_obj);
-				elseif ($node->tagName == 'radiogroup') $this->_transform_radiogroup($node, $disco_obj);
-				elseif ($node->tagName == 'checkboxgroup') $this->_transform_checkboxgroup($node, $disco_obj);
-				elseif ($node->tagName == 'optiongroup') $this->_transform_optiongroup($node, $disco_obj);
-				elseif ($node->tagName == 'hidden') $this->_transform_hidden($node, $disco_obj);
-				elseif ($node->tagName == 'comment') $this->_transform_comment($node, $disco_obj);
-				elseif ($node->tagName == 'upload') {
-					$disco_obj->form_enctype = "multipart/form-data";
-					$this->_transform_upload($node, $disco_obj);
-				} elseif ($node->tagName == 'event_tickets') {
-					$this->_transform_event_tickets($node, $disco_obj);
+				if (!$node->tagAttrs['deleted']) {
```

Comment:
There's a PHP notice that pops up on forms that exist today and don't have the deleted attr.
```
Notice: Undefined index: deleted at …/thor/thor.php:184
```
Could you tweak the line so it ensures the key 'deleted' exists in tagAttrs, first?